src/components/App.js

```diff
@@ -76,42 +66,28 @@ const App = () => {
 
     const eventRelay = useSelector((state) => state.services.eventRelay);
     const eventRelayUrl = eventRelay?.url ?? null;
-    const openIdConfig = useSelector((state) => state.openIdConfiguration.data);
 
     const [lastIsAuthenticated, setLastIsAuthenticated] = useState(false);
 
     const [didPostLoadEffects, setDidPostLoadEffects] = useState(false);
 
-    const isInAuthPopup = (() => {
-        try {
-            const didCreateSignInPopup = localStorage.getItem(LS_SIGN_IN_POPUP);
-            return (
-                window.opener && window.opener.origin === BENTO_URL_NO_TRAILING_SLASH && didCreateSignInPopup === "true"
-            );
-        } catch {
-            // If we are restricted from accessing the opener, we are not in an auth popup.
-            return false;
-        }
-    })();
+    const isInAuthPopup = checkIsInAuthPopup(BENTO_URL_NO_TRAILING_SLASH);
+
+    const openIdConfig = useOpenIdConfig(OPENID_CONFIG_URL);
 
     // Set up auth callback handling
-    useHandleCallback(CALLBACK_PATH, isInAuthPopup ? popupOpenerAuthCallback : undefined);
+    useHandleCallback(
+        CALLBACK_PATH,
+        fetchUserDependentData,
```

Comment:
having the `nop` here wasn't redundant - it was needed since fetchUserDependentData in bento web is an action factory, not an action itself

Reply:
`fetchUserDependentData` accepts undefined `servicesCb`, in which case it will use `nop` inside the thunk. Or am I missing something?

Reply:
then it should be `fetchUserDependentData()` not `fetchUserDependentData`, since `fetchUserDependentData` is a factory that produces an action (and I think `useHandleCallback` expects an action, not an action factory)